niceml/utilities/readwritelock.py

```diff
@@ -168,7 +179,7 @@ def release(self):
 def is_lock_file_acquirable(
     lock_file_path: str, file_system: Optional[AbstractFileSystem] = None
 ) -> bool:
-    """Check if the lock file is available."""
+    """Check if the lock file is available (== not existing at lock_file_path)."""
```

Comment:
Is this supposed to be part of the doc string?

Reply:
Yes, to explain what availablity in this case means. Do you have a better suggestion of the docstring?

Reply:
I am not sure if I have understood this correctly. 
I would suggest something like "Check if the lock file is available. The lock file is available if it does not exist at lock_file_path."

Reply:
I think you understood it very well. I changed the docstring according to your suggestion.

Reply:
Perfekt. Thank you.